epamcscockpit/web/src/com/epam/helper/RestHelper.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.helper;
+
+import com.epam.service.RestAuthServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.web.client.RestTemplate;
+
+public class RestHelper {
```

Comment:
add java doc  understanding purpose of class. Method "call" in class "Helper" says nothing